packages/ui-components/src/components/accordiontoolbar.ts

```diff
@@ -63,6 +63,15 @@ class AccordionToolbarLayout extends AccordionLayout {
     }
   }
 
+  /**
+   * Function is called when updating the title. Updating the title build a new Title
+   * widget, we need to attach again the toolbar node to it.
```

Comment:
The "Updating the title build a new Title widget" part of the comment is not clear to me, it looks like it might be an editing error?

Reply:
Thanks for you review @krassowski.
I agree that the comment was unclear, I simplified it.